### ISSUE TITLE
Better error in CLI when URL cannot be parsed on login

### ISSUE THREAD

**Issue by superseb** [rancher/rancher#14520]
Old:

```
./rancher login --token bla:ba 127.0.0.1
FATA[0000] Get /v3: unsupported protocol scheme ""
```
New:

```
# ./rancher login --token bla:ba 127.0.0.1
FATA[0000] Failed to parse SERVERURL (127.0.0.1), make sure it is a valid HTTPS URL (e.g. https://rancher.yourdomain.com or https://1.1.1.1). Error: parse 127.0.0.1: invalid URI for request
```

**Comment by deniseschannon** [comment]
Available in v2.0.4-rc1

**Comment by moelsayed** [comment]
Validated using `v2.0.4-rc1`. Error message showed as expected.
